WebRoot/js/zimbraAdmin/common/ZaItem.js

```diff
@@ -115,6 +115,32 @@ function(a,b) {
 	return ZaItem.compareAttr(a,b,"description");
 }
 
+ZaItem.compareEmailAddressAsc = 
+function(a,b) {
+	var al = a.emailAddress.toLowerCase();
+	var bl = b.emailAddress.toLowerCase();
+
+	if (al < bl)
```

Comment:
use curly braces to conditional statements

Reply:
updated